dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/utils.py

```diff
@@ -856,6 +856,18 @@ def transform_atan3_operator_evalf(line):
 
     return line_tmp_bis
 
+##
+# Transform _event_floor(x, index, data) to (modelica_integer)floor(x)
+# @param line : line to analyse
+# @return line transformed
+def replace_event_floor(line):
+    if "_event_floor" not in line:
```

Comment:
according to the update of the FixedBooleanDelay Model, _event_integer should also be added